pkg/webui/lib/shared-messages.js

```diff
@@ -121,6 +121,7 @@ export default defineMessages({
   deviceDescPlaceholder: 'Description for my new end device',
   deviceIdPlaceholder: 'my-new-device',
   deviceNamePlaceholder: 'My new end device',
+  deviceSimulationDisabledWarning: 'Simulation is disabled for devices that skip payload crypto.',
```

Comment:
```suggestion
  deviceSimulationDisabledWarning: 'Simulation is disabled for devices that skip payload crypto',
```

[https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.13/DESIGN.md#general-considerations](https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.13/DESIGN.md#general-considerations)

> *   Do not use end-of-sentence periods, unless the text consists of multiple sentences